src/main/java/com/faforever/client/chat/OfficialLanguageChannel.java

```diff
@@ -0,0 +1,35 @@
+package com.faforever.client.chat;
+
+import java.util.Optional;
+
+public enum OfficialLanguageChannel {
+  FRENCH("fr", "#french"),
+  GERMAN("de", "#german"),
+  RUSSIAN("ru", "#russian");
+
+  private final String localeLanguage;
+  private final String channelName;
+
+
+  OfficialLanguageChannel(String localeLanguage, String channelName) {
+    this.localeLanguage = localeLanguage;
+    this.channelName = channelName;
+  }
+
+  public static Optional<OfficialLanguageChannel> getChannelName(String localeLanguage) {
+    for (OfficialLanguageChannel channel : values()) {
+      if (channel.getLocaleLanguage().equals(localeLanguage)) {
+        return Optional.of(channel);
+      }
+    }
+    return Optional.empty();
+  }
+
+  public String getChannelName() {
```

Comment:
Use Lomboks @Getter on class level - makes the getter obsolete

Reply:
Im in favor of removing getters entirely. I don't see why they would be needed. static method `getChannelName` can just return the channelName directly, as its name implies.